sync_diff_inspector/utils/utils.go

```diff
@@ -35,6 +35,18 @@ import (
 	"go.uber.org/zap"
 )
 
+// SecretString is a wrapper for sensitive strings like password,
+// which yields redacted string when being marshaled.
+type SecretString string
+
+func (s SecretString) MarshalJSON() ([]byte, error) {
+	return []byte(`"******"`), nil
+}
+
+func (s SecretString) String() string {
+	return "******"
```

Comment:
we need use it to generate DSN, is it ok to use `******`?
https://github.com/pingcap/tidb-tools/blob/422258e0d5cd216d3dafb6670dee0c93be269120/sync_diff_inspector/config/config.go#L176

Reply:
I think perhaps we'd better manually unwrap the secret string here 🤔